pkg/builds/schema_doc.go

```diff
@@ -486,6 +487,12 @@ var generatedSchemaJSON = `{
           "title": "exoscale",
           "$ref": "#/definitions/artifact"
         },
+        "oscontainer": {
+          "$id": "#/properties/images/properties/oscontainer",
+          "type": "object",
+          "title": "oscontainer",
+          "$ref": "#/definitions/artifact"
+        },
```

Comment:
Based on https://github.com/coreos/coreos-assembler/pull/3111#issuecomment-1274818871 I would have expected this entry to be named `legacy-oscontainer`.

Reply:
I am very confused. Meeting?